### ISSUE TITLE
ARM platform support: pyrfc import error on MacBook M1

### ISSUE THREAD

**Issue by ElGarno** [SAP/PyRFC#278]
**Describe the bug**
I have a problem using the pyrfc (2.5.0) library to collect data from SAP. I installed this library via pycharm and also manually with pip and in both cases the installation is successful.

Problem arises when importing the libraby: 
**dlopen(/.../venv/lib/python3.10/site-packages/pyrfc/pyrfc.cpython-310-darwin.so, 0x0002): symbol not found in flat namespace '_RfcAppendNewRow'**

I use a MacBook Pro M1, Python 3.10 (as you can see) and also tried to use pynwrfc but nothing worked. Any ideas?

**To Reproduce**
I installed the library as described on http://sap.github.io/PyRFC/install.html --> Mac

**Environment**
 - OS: MacOS Monterey 12.3.1
 - PyRFC version 2.5.0

Thanks for helping!

**Comment by bsrdjan** [comment]
SAP NWRFC SDK is not supported on ARM platform and connectors based on RFC SDK are not supported on ARM:
- https://launchpad.support.sap.com/#/notes/2573790
- https://github.com/SAP/node-rfc/issues/249
The feature request can be created for SAP NWRFC SDK

**Comment by bsrdjan** [comment]
:tada: ARM is now supported, using PyRFC 2.7.1 and SAP NW RFC SDK 7.50.11 :tada:

**Comment by melquic** [comment]
Hello,

I have been trying to install the SAP NWRFC SDK on my MacBook M1, but I am facing an issue. Below are the steps I have followed so far:

I downloaded the SAP NWRFC SDK zip file (SAP_NetWeaver_RFC_SDK_750 (PL 11).zip) and unzipped it using the unzip command.

Following the provided installation instructions, I added the nwrfcsdk/lib directory to the DYLD_LIBRARY_PATH environment variable by adding the following line to my ~/.zshrc file:

`export DYLD_LIBRARY_PATH=$DYLD_LIBRARY_PATH:/path/to/nwrfcsdk/lib`

I also added the nwrfcsdk/bin directory to the PATH environment variable:
`export PATH=$PATH:/path/to/nwrfcsdk/bin`

After saving the changes and restarting the terminal, I set the SAPNWRFC_HOME environment variable in my ~/.zshrc file:
`export SAPNWRFC_HOME=/path/to/nwrfcsdk`

I then attempted to install pyrfc, using Poetry. However, during the installation process, I encountered the following error:

![image](https://user-images.githubusercontent.com/1239634/232891172-8b9b021f-ed3b-42ba-9163-2be750af2608.png)

Unfortunately, there is no libsapnwrfc.dylib file in the lib directory, and I am unsure of how to proceed. I would appreciate any assistance or guidance on how to successfully install the SAP NWRFC SDK on my MacBook M1 and resolve the aforementioned issue.

My Machine Details:
Macbook Pro 2021
Chip Apple M1 Pro
macOS monterey: Version 12.6

Thank you in advance for your help!
